components/application/index.jsx

```diff
@@ -21,6 +22,7 @@ const Application = ({
       <AppBar isAuthorized={isAuthorized}>
         {dataProvider && <RepositorySelect value={dataProvider} />}
         <Logout />
+        <LoadingBar />
```

Comment:
Even if the loading bar is on top of application bar, it has nothing to the application bar.

Reply:
It's more about positioning and I liked it more having it below App bar.